lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -573,6 +573,12 @@ void SelectionDAG::RemoveDeadNodes(SmallVectorImpl<SDNode *> &DeadNodes) {
   while (!DeadNodes.empty()) {
     SDNode *N = DeadNodes.pop_back_val();
 
```

Comment:
This is needed to fix an assertion error in Chrome. Caroline found this could fix that problem. https://reviews.llvm.org/D33731